app/models/plan.rb

```diff
@@ -320,6 +348,17 @@ def owned_by?(user_id)
   end
 
   ##
+  # determines if the plan is reviewable by the specified user
+  #
+  # @param user_id [Integer] the id for the user
+  # @return [Boolean] true if the user can administer the plan
+  def reviewable_by?(user_id)
+    user_id = user_id.id if user_id.is_a?(User)
```

Comment:
user_id I believe is an integer so for never will be an object of class User

Reply:
true, should be. I copied it from other ones Sam created like `administerable_by?` etc. Assumed that he did that because there was a situation where the User object was being supplied instead of the id. Going to leave it as is for now. We should change all of them if they aren't needed